plugins/zoom-to-fit/README.md

```diff
@@ -0,0 +1,52 @@
+# @blockly/zoom-to-fit [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) plugin that adds a
+zoom-to-fit control to the workspace.
+
+## Installation
+
+### Yarn
+```
+yarn add @blockly/zoom-to-fit
+```
+
+### npm
+```
+npm install @blockly/zoom-to-fit --save
+```
+
+## Usage
+
+```js
+import * as Blockly from 'blockly';
+import {ZoomToFitControl} from '@blockly/zoom-to-fit';
+
+// Inject Blockly.
+const workspace = Blockly.inject('blocklyDiv', {
+  toolbox: toolboxCategories,
+});
+
+// Initialize plugin.
+const zoomToFit = new ZoomToFitControl(workspace);
+zoomToFit.init();
+workspace.getPluginManager().addPlugin({
```

Comment:
Let's have a chat as a team about how we want to add plugins to the workspace (like this vs in the config struct).

Reply:
Discussed and `addPlugin` call was moved inside of `init` since we may be changing the api in the near future.